src/types.js

```diff
@@ -23,5 +23,11 @@
  * @prop {import('preact').Ref<HTMLElement>} [elementRef]
  */
 
+/**
+ * A type describing any of the standalone icon components, which take any
+ * valid SVG element attribute as props.
```

Comment:
```suggestion
 * valid `<svg>` element attribute as props.
```

The type specifically refers to the `<svg>` element, rather than *any* element from the SVG namespace / that can occur inside an `<svg>` element.